pkgs/build-support/node/fetch-npm-deps/default.nix

```diff
@@ -0,0 +1,111 @@
+{ lib, stdenv, rustPlatform, Security, testers, fetchurl }:
+
+rec {
```

Comment:
It's not nice that these reference each other, because it means when I change fetch-npm-deps in an overlay fetchNpmDeps will still use the old version.

Better take fetch-npm-deps as an input in line 1, so that it is provided by callPackage.